server/controllers/auth/login.js

```diff
@@ -10,8 +9,7 @@ exports.login = (req, res, next) => {
   const key = process.env.KEY;
   let id;
   let user;
-  loginSchema.validate(req.body)
-    .then(() => getUserByUsername(username))
+  getUserByUsername(username)
```

Comment:
**_why_** this file here ?!!!!!!

Reply:
and in the pervious merged branches, 
you put the login schema and here you solved the conflict by removing it 

this is the PR you add the login schema on it #18